### PR TITLE
Push directly to cgr, remove monopod replaces

### PR DESCRIPTION
**Pull request by jdolitsky**

### CODE REVIEW

File: .github/actions/release-image/action.yml [hunk 1]
```diff
@@ -320,11 +326,6 @@ runs:
               tag="${tag}${{ inputs.apkoTargetTagSuffix }}"
             fi
             crane cp "${REF}" "${tag}"
-            # For public images, copy them over to Chainguard registry
```
[the remark this answers is not in the document]
Comment (jdolitsky) [reply .github/actions/release-image/action.yml:323]: oh youre saying do the inverse?

[the remark this answers is not in the document]
Comment (mattmoor) [reply .github/actions/release-image/action.yml:323]: Yeah basically flip it around

[the remark this answers is not in the document]
Comment (jdolitsky) [reply .github/actions/release-image/action.yml:323]: 🫡

[the remark this answers is not in the document]
Comment (jdolitsky) [reply .github/actions/release-image/action.yml:323]: done

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -91,14 +92,6 @@ jobs:
...
           done
           echo "release-image-inputs=$(cat inputs.json | tr -d '\n')" >> $GITHUB_OUTPUT
-      # Sleep a random amount of time up to 10 minutes.
-      # This staggers our pushes, to hopefully avoid hitting rate limits.
-      # See https://github.com/chainguard-images/images/issues/173
-      - name: Random Sleep
-        run: |
-          rand=$(( ( RANDOM % 600 )  + 1 ))
-          echo "sleeping $rand seconds..."
-          sleep $rand
```
Comment (imjasonh) [inline .github/workflows/release.yaml:101]: We'll still need this if we're pushing to GHCR, or else the workflows will fail :(

Comment (jdolitsky) [reply .github/workflows/release.yaml:101]: ill put the sleep right before this tagging process

File: .github/actions/release-image/action.yml [hunk 3]
```diff
@@ -58,6 +58,13 @@ runs:
         archs: ${{ inputs.melangeArchs }}
         template: ${{ inputs.melangeTemplate }}
 
+    # If publishing to Chainguard registry (cgr.dev), setup auth
+    - id: cgrauth1
+      if: contains(inputs.apkoBaseTag, 'cgr.dev/') && inputs.chainguardIdentity != ''
```
Comment (mattmoor) [inline .github/actions/release-image/action.yml:63]: We should just do this whenever there's an identity IMO

**Review by mattmoor — APPROVED** [review]

File: .github/actions/release-image/action.yml (hunk 3 above)
Comment (jdolitsky) [reply .github/actions/release-image/action.yml:63]: we can add back if needed, but for some matrix legs in other repos, may not be doing anything related to cgr.dev

**Pull request merged**